pss/privileged-containers/policy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: wildcard_images_violate_true_image_not_match_wildcard
+spec:
+  validationFailureAction: enforce
+  rules:
+  - name: wildcard_images_violate_true_image_not_match_wildcard
+    match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    validate:
+      podSecurity:
+        level: baseline
+        version: v1.24
+        exclude:
+        - controlName: Privileged Containers
+          images:
+          - nginx*
```

Comment:
You're writing the wildcard capability but you aren't testing for it. Please add a test to explicitly check this code path.

Reply:
Without `images` the checks are performed at the spec level.

Reply:
So in this case for Privileged Containers, is defining one or more images required because it has no `spec` fields to enforce?

Reply:
Yes, I'm adding the validation checks in.

Reply:
Does `images` allow a single wildcard character to match on all image names?

Reply:
Yes, wildcard match is supported.